### DISCUSSION TITLE
access to folders not allowed

### DISCUSSION THREAD

**Discussion by alesandrorc** [requarks/wiki#6648] [Error / Bug Report]
EN
I separated folders by groups, and it works fine. But when I log out of the user and it is on rules, it lists all the folders that were not supposed to have permission. And without being logged in with any user.
PT-BR
Separei pastas por grupos, e funciona certo. Porem quando faço logout do usuario e esta sobre regras, ele lista todas as pastas que não era para ter permissão. E sem estar logado com qualquer usuario.
With user logged in
![image](https://github.com/requarks/wiki/assets/31317055/4f085929-bff9-4a5c-96c1-12438a96930d)

when you click on exiting the user it shows all the folders that were not supposed to have permission
![image](https://github.com/requarks/wiki/assets/31317055/12343966-ed83-43af-b660-04a2b2d77686)
Wiki.js 2.5.201

**Comment by NGPixel** [comment]
Did you set the necessary rules on the Guests group?

**Comment by alesandrorc** [comment]
The group only has access to one folder rejecting the others. When logged in it works, but when I go to ACCOUNT > EXIT , it lists all folders for a guest user, I think...
![image](https://github.com/requarks/wiki/assets/31317055/a7e274f1-5eee-4c99-a795-92023b8bb4d0)


In the second image, note that no one is logged in, but he has given access to all folders

**Comment by NGPixel** [comment]
Do you have users in multiple groups?
